<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>781</width>
      <height>541</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(85, 0, 127);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>40</y>
       <width>211</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>LISTA DE PRODUTOS</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="tableWidget">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>90</y>
       <width>511</width>
       <height>381</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(85, 0, 127);
background-color: rgb(255, 255, 255);</string>
     </property>
     <column>
      <property name="text">
       <string>ID</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>CÓDIGO</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>DESCRIÇÃO</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>PREÇO</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>CATEGORIA</string>
      </property>
     </column>
    </widget>
    <widget class="QPushButton" name="cancelar">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>500</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);
border-radius: 10px;</string>
     </property>
     <property name="text">
      <string>CANCELAR</string>
     </property>
    </widget>
    <widget class="QPushButton" name="deletar">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>500</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);
border-radius: 10px;</string>
     </property>
     <property name="text">
      <string>DELETAR</string>
     </property>
    </widget>
    <widget class="QPushButton" name="editar">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>500</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);
border-radius: 10px;</string>
     </property>
     <property name="text">
      <string>EDITAR</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
